src/components/DashboardAlphabet/DashboardAlphabetPresentation.js

```diff
@@ -89,7 +90,7 @@ function DashboardAlphabetPresentation({
               </td>
               <td className="px-1 py-4 whitespace-nowrap text-right text-sm font-medium">
                 <Link
-                  to={`/${sitename}/alphabet/${character?.title}`}
+                  to={`/${sitename}/alphabet?${CHAR}=${character?.title}`}
```

Comment:
Needs "startsWith/"... or maybe this way is more concise and the other ones should match this?

Reply:
Good eye 👍  
We have two routes `alphabet?${CHAR}=` which is for the alphabet page itself with character details
and `alphabet/startsWith?${CHAR}=` which is for the list of dictionary entries starting with the character.

I decided this link in the dashboard actually made more sense to link to the alphabet page because it relates more closely to what is edited in the Dashboard so it is switching over to the other route.